rpcclient/local.go

```diff
@@ -0,0 +1,216 @@
+package rpcclient
+
+import (
+	"context"
+	"fmt"
+	tmbytes "github.com/lazyledger/lazyledger-core/libs/bytes"
+	tmpubsub "github.com/lazyledger/lazyledger-core/libs/pubsub"
+	tmquery "github.com/lazyledger/lazyledger-core/libs/pubsub/query"
+	rpcclient "github.com/lazyledger/lazyledger-core/rpc/client"
+	"github.com/lazyledger/lazyledger-core/rpc/core"
+	ctypes "github.com/lazyledger/lazyledger-core/rpc/core/types"
+	rpctypes "github.com/lazyledger/lazyledger-core/rpc/jsonrpc/types"
+	"github.com/lazyledger/lazyledger-core/types"
+	"github.com/lazyledger/optimint/node"
+	"time"
+)
```

Comment:
nit: sort the imports into blocks: one for the standard lib, one for external libs and one for application libs

Reply:
I'm used to use vim, and it handles this in a pretty way. Now I'm trying to switch to goland, and imports are folded in editor (so I pay less attention), and the order is crazy after "add missing import". Definitely will fix this!